recipes/argopt/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "argopt" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c8ca9cdeb6bf5e8d6b0fb6a6cd3c676210fd3ebacdabab57f5b95d1ed514f0af
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - argopt
+  requires:
```

Comment:
You can remove the `requires` section, because you aren't running any of the tests that requires those dependencies.

Reply:
Removed in new commit